ui/listener-api/src/main/scala/pl/touk/nussknacker/ui/listener/services/PullProcessRepository.scala

```diff
@@ -0,0 +1,15 @@
+package pl.touk.nussknacker.ui.listener.services
+
+import pl.touk.nussknacker.restmodel.process.ProcessId
+import pl.touk.nussknacker.restmodel.processdetails.BaseProcessDetails
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+
+import scala.concurrent.ExecutionContext
+
+trait PullProcessRepository[F[_]] {
+  def fetchLatestProcessDetailsForProcessId(id: ProcessId)
```

Comment:
I wonder if it makes sense to abstract over F? Will we have other implementation in the future? 
And also if we should pass LoggedUser from plugin? Maybe it'd be better to pass some internal user in NK itself?

Reply:
You're right - it makes no sense.
I like the idea of deidcated user for these kind of actions. Check out the code now :)